components/feature/downloads/src/test/java/mozilla/components/feature/downloads/DownloadDialogFragmentTest.kt

```diff
@@ -22,10 +22,12 @@ class DownloadDialogFragmentTest {
     @Before
     fun setup() {
         dialog = object : DownloadDialogFragment() {}
+        val currentBytes = ("Mozilla/5.0 (Linux; Android 7.1.1) AppleWebKit/537.36 (KHTML" +
+                ", like Gecko) Version/4.0 Focus/8.0 Chrome/69.0.3497.100 Mobile Safari/537.36").length.toLong()
         download = DownloadState(
             "http://ipv4.download.thinkbroadband.com/5MB.zip",
             "5MB.zip", "application/zip", 5242880,
-            "Mozilla/5.0 (Linux; Android 7.1.1) AppleWebKit/537.36 (KHTML, like Gecko) Version/4.0 Focus/8.0 Chrome/69.0.3497.100 Mobile Safari/537.36"
+                currentBytes
```

Comment:
Nit: I think this value was for the user agent, the `currentBytesCopied` is optional, we are not required to pass a value. 
`                userAgent = "Mozilla/5.0 (Linux; Android 7.1.1) AppleWebKit/537.36 (KHTML, like Gecko) Version/4.0 Focus/8.0 Chrome/69.0.3497.100 Mobile Safari/537.36"`